src/main/java/com/trainigcenter/springtask/web/controller/ActorController.java

```diff
@@ -0,0 +1,121 @@
+package com.trainigcenter.springtask.web.controller;
+
+import com.trainigcenter.springtask.domain.Actor;
+import com.trainigcenter.springtask.service.ActorService;
+import com.trainigcenter.springtask.web.dto.ActorDto;
+import com.trainigcenter.springtask.web.dto.Error;
+import com.trainigcenter.springtask.web.exception.ForbiddenException;
+import com.trainigcenter.springtask.web.exception.NotFoundException;
+import com.trainigcenter.springtask.web.exception.WebException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/actors")
+public class ActorController {
+
+    private static final Logger logger = LogManager.getLogger(ActorController.class);
+
+    private final ActorService actorService;
+    private final ModelMapper modelMapper;
+
+    @Autowired
+    public ActorController(ActorService actorService, ModelMapper modelMapper) {
+        this.actorService = actorService;
+        this.modelMapper = modelMapper;
+    }
+
+    @GetMapping
+    public List<ActorDto> getAll() {
+        List<Actor> allActors = actorService.getAll();
+        return allActors.stream()
+                        .map(this::convertToDto)
+                        .collect(Collectors.toList());
+    }
+
+    @GetMapping("/{id}")
+    public ActorDto getActorById(@PathVariable Integer id) {
+        Actor actor = Optional.ofNullable(actorService.getById(id))
+                              .orElseThrow(() -> new NotFoundException("Actor id:" + id + " not found"));
+
+        return convertToDto(actor);
```

Comment:
The method body can be combined into one chain call. Valid for all similar cases for the entire application.